src/home/searchbar.js

```diff
@@ -3,17 +3,16 @@ import React from "react";
 const Searchbar = () => {
   return (
     <>
-      <div className="row mb-2">
-        <div className="col-11 position-relative">
+      <div className="mb-3">
           <input
             placeholder="Search eBuy"
             className="form-control rounded-pill ps-5"
+            style={{width: "100%", backgroundColor: "#1E1E1E", color: "#FFFFFF"}}
           />
           <i
-            className="bi bi-search position-absolute
-                       wd-nudge-up"
-          ></i>
-        </div>
+            className="bi bi-search position-relative"
```

Comment:
not a big deal but the search magnifying glass icon is not centre aligned with the searchbar

Reply:
idk how to center align it 🥲